docs/hooks/medication-prescribe.md

```diff
@@ -19,131 +19,279 @@ Field | Optionality | Prefetch Token | Type | Description
+            },
+            "status":"draft",
+            "intent":"order",
+            "medicationCodeableConcept":{
+              "coding":[
+                {
+                  "system":"http://www.nlm.nih.gov/research/umls/rxnorm",
+                  "code":"617993",
+                  "display":"Amoxicillin 120 MG/ML / clavulanate potassium 8.58 MG/ML Oral Suspension"
+                }
+              ],
+              "text":"Amoxicillin 120 MG/ML / clavulanate potassium 8.58 MG/ML Oral Suspension"
+            },
+            "subject":{
+              "reference":"Patient/1288992"
+            },
+            "dosageInstruction":[
               {
-                "system":"http://www.nlm.nih.gov/research/umls/rxnorm",
-                "code":"617993",
-                "display":"Amoxicillin 120 MG/ML / clavulanate potassium 8.58 MG/ML Oral Suspension"
+                "text":"5 mL bid x 10 days",
+                "timing":{
+                  "repeat":{
+                    "boundsPeriod":{
+                      "start":"2005-01-04"
+                    },
+                    "frequency":2,
+                    "period":1,
+                    "periodUnit":"d"
+                  }
+                },
+                "doseQuantity":{
+                  "value":5,
+                  "unit":"mL",
+                  "system":"http://unitsofmeasure.org",
+                  "code":"mL"
+                }
               }
             ],
-            "text":"Amoxicillin 120 MG/ML / clavulanate potassium 8.58 MG/ML Oral Suspension"
-          },
-          "dosageInstruction":[
-            {
-              "text":"5 mL bid x 10 days",
-              "timing":{
-                "repeat":{
-                  "boundsPeriod":{
-                    "start":"2005-01-04"
-                  },
-                  "frequency":2,
-                  "period":1,
-                  "periodUnits":"d"
-                }
-              },
-              "doseQuantity":{
-                "value":5,
+            "dispenseRequest":{
+              "numberOfRepeatsAllowed":1,
+              "quantity":{
+                "value":1,
                 "unit":"mL",
                 "system":"http://unitsofmeasure.org",
                 "code":"mL"
+              },
+              "expectedSupplyDuration":{
+                "value":10,
+                "unit":"days",
+                "system":"http://unitsofmeasure.org",
+                "code":"d"
               }
             }
-          ],
-          "dispenseRequest":{
-            "numberOfRepeatsAllowed":1,
-            "quantity":{
-              "value":1,
-              "unit":"mL",
-              "system":"http://unitsofmeasure.org",
-              "code":"mL"
+          }
+        },
+        {
+          "resource":{
+            "resourceType":"MedicationRequest",
+            "id":"smart-MedicationRequest-104",
+            "meta":{
+              "lastUpdated":"2018-04-30T13:26:48.124-04:00",
```

Comment:
This trailing comma makes the JSON malformed. You can test this by pasting the JSON into a linter/formatter like https://jsonformatter.curiousconcept.com/